tests/prague/eip7692_eof_v1/eip3540_eof_v1/test_section_size.py

```diff
@@ -0,0 +1,97 @@
+"""
+EOF Container, test custom_size field for sections
+"""
+
+from enum import Enum
+
+import pytest
+
+from ethereum_test_tools import EOFTestFiller
+from ethereum_test_tools import Opcodes as Op
+from ethereum_test_tools.eof.v1 import Container, EOFException, Section, SectionKind
+
+from .. import EOF_FORK_NAME
+
+REFERENCE_SPEC_GIT_PATH = "EIPS/eip-3540.md"
+REFERENCE_SPEC_VERSION = "8dcb0a8c1c0102c87224308028632cc986a61183"
+
+pytestmark = pytest.mark.valid_from(EOF_FORK_NAME)
+
+
+class SectionSize(Enum):
+    """
+    Enum for the section size
+    """
+
+    NORMAL = 1
+    UNDERSIZE = 2
+    OVERSIZE = 3
+
```

Comment:
```suggestion
    def __str__(self) -> str:
        """
        Returns the string representation of the section kind
        """
        return self.name

```
Adding this `str` function makes it so that the pytest id name is a bit easier to read.

Reply:
Actually it would be nice to add a similar function to the `SectionKind` enum in `src/ethereum_test_tools/eof/v1/__init__.py`:
```python
class SectionKind(IntEnum):
    """
    Enum class of V1 valid section kind values
    """

    TYPE = 1
    CODE = 2
    CONTAINER = 3
    DATA = 4

    def __str__(self) -> str:
        """
        Returns the string representation of the section kind
        """
        return self.name
```

It allows printing the actual section type name into the test ID.

Reply:
thats why I used Enum, with enum it comes automatically